closed/OpenJ9.gmk

```diff
@@ -315,6 +322,13 @@ endif
 ifneq (,$(OPENJ9_DEVELOPER_DIR))
   CUSTOM_COMPILER_ENV_VARS += DEVELOPER_DIR="$(OPENJ9_DEVELOPER_DIR)"
 endif
+ifneq (true,$(OPENJ9_ENABLE_CMAKE))
+  ifeq (true,$(OPENJ9_ENABLE_INLINETYPES))
+    CMAKE_ARGS += J9VM_OPT_VALHALLA_VALUE_TYPES=ON
+  else # OPENJ9_ENABLE_INLINETYPES
+    CMAKE_ARGS += J9VM_OPT_VALHALLA_VALUE_TYPES=OFF
```

Comment:
These need to be `-DJ9VM_OPT_VALHALLA_VALUE_TYPES=...` (`-D` is missing).
Also, `CMAKE_ARGS` needs to actually be used.